CHANGELOG

```diff
@@ -1,3 +1,8 @@
+Unreleased
+----------
+* rename WebhooksController module to WebhookVerification
```

Comment:
it is, WebhooksController was only added in a recent version (6.3.0) so not sure how many people have adopted it yet. I'll update the note and when we release we'll bump to 6.5.0